src/App.tsx

```diff
@@ -15,48 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
```

Comment:
You could've add the initial state as None  = '' here and use it later as default for sortField.